greek/marshmallow.md

```diff
@@ -23,4 +23,5 @@
 [Προσφέρεις στο ζαχαρωτό τσιγάρο](offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md)
 
 Τελικά συνειδητοποιείς ότι ο Stay Puft Marshmallow Man από το Ghostbusters ήταν στη πραγματικότητα το 
-ζαχαρωτό και σε πιάνει τρόμος ... 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.